templates/single.twig

```diff
@@ -6,7 +6,7 @@
         <div class="col-md-12 post-image" style="background-image:url({{post.thumbnail.src('large')}})"></div>
         <div class="col-sm-offset-1 col-md-offset-1 col-xs-12 col-sm-10 col-md-10 article-body">
           <h1>{{post.title}}</h1>
-          <p class="article-meta">{{post.post_date|date('j F Y')}} · by {{post.author.name}}</p>
```

Comment:
How does it work with an empty string ? Say we revert the author

Reply:
What scenario is there an empty string? Posts have to be associated with an author which will have a name. If you want to get rid of the guest author, you just delete the custom field

Reply:
Cool then, I didn't know that deleting a custom field was straightforward